meteor-testapp/package.json

```diff
@@ -5,6 +5,7 @@
     "start": "meteor run"
   },
   "dependencies": {
+    "@babel/runtime": "^7.8.4",
```

Comment:
Hm, should we have both `@babel/runtime` and `babel-runtime`? Not sure if there's actually a difference or if this is just a duplicate.

Reply:
IIRC if I tried removing either one, it broke... but I have no idea, feel free to play with it further.